From petkan@nucleusys.com Mon Dec  4 04:26:42 2006
Date: Mon, 4 Dec 2006 14:27:36 +0200 (EET)
From: Petko Manolov <petkan@nucleusys.com>
To: gregkh@suse.de
Cc: <N.E.Whiteford@soton.ac.uk>
Subject: USB: rtl8150 new device id
Message-ID: <Pine.LNX.4.64.0612041423070.17432@localhost.localdomain>

This one adds another vendor ID to rtl8150 driver.  Please apply.


Signed-off-by: Petko Manolov <petkan@nucleusys.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/net/rtl8150.c |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

--- gregkh-2.6.orig/drivers/usb/net/rtl8150.c
+++ gregkh-2.6/drivers/usb/net/rtl8150.c
@@ -124,10 +124,11 @@
 #define	RX_URB_FAIL		3
 
 /* Define these values to match your device */
-#define VENDOR_ID_REALTEK		0x0bda
+#define	VENDOR_ID_REALTEK		0x0bda
 #define	VENDOR_ID_MELCO			0x0411
-#define VENDOR_ID_MICRONET		0x3980
+#define	VENDOR_ID_MICRONET		0x3980
 #define	VENDOR_ID_LONGSHINE		0x07b8
+#define	VENDOR_ID_OQO			0x1557
 #define	VENDOR_ID_ZYXEL			0x0586
 
 #define PRODUCT_ID_RTL8150		0x8150
@@ -144,6 +145,7 @@ static struct usb_device_id rtl8150_tabl
 	{USB_DEVICE(VENDOR_ID_MELCO, PRODUCT_ID_LUAKTX)},
 	{USB_DEVICE(VENDOR_ID_MICRONET, PRODUCT_ID_SP128AR)},
 	{USB_DEVICE(VENDOR_ID_LONGSHINE, PRODUCT_ID_LCS8138TX)},
+	{USB_DEVICE(VENDOR_ID_OQO, PRODUCT_ID_RTL8150)},
 	{USB_DEVICE(VENDOR_ID_ZYXEL, PRODUCT_ID_PRESTIGE)},
 	{}
 };
